pkg/tsdb/cloudwatch/log_query.go

```diff
@@ -96,9 +96,22 @@ func logsResultsToDataframes(response *cloudwatchlogs.GetQueryResultsOutput) (*d
 
 	frame := data.NewFrame("CloudWatchLogsResponse", newFields...)
 	frame.Meta = &data.FrameMeta{
+		Stats: []data.QueryStat{
+			{
+				FieldConfig: data.FieldConfig{DisplayName: "Bytes scanned"},
+				Value:       *response.Statistics.BytesScanned,
```

Comment:
If any of these floats are nil, then de-referencing them will cause a panic.  So it needs to be checked that that value is not nil first.

Other than that looks good to me.

Reply:
Agree with Kyle.